src/core/components/operations.jsx

```diff
@@ -28,92 +27,89 @@ export default class Operations extends React.Component {
   render() {
     let {
       specSelectors,
+    } = this.props
+
+    // Note returns now optional filtered and optional hierarchical structure, in case user has set config accordingly
+    const taggedOps = specSelectors.taggedOperations()
```

Comment:
I like this approach! This makes a lot of sense to me. That said, it also introduces a major, backward-incompatible change, so it may not fly.... As with the rest of this, though, I say let's go for it and see what Tim thinks. If he asks us to factor it into a hierarchy-specific selector, we can do that then.

(This is not a blocker, just an idle comment.)